homeassistant/components/nws/const.py

```diff
@@ -169,6 +179,7 @@ class NWSSensorEntityDescription(SensorEntityDescription):
         name="Wind Direction",
         icon="mdi:compass-rose",
         device_class=None,
+        state_class=STATE_CLASS_MEASUREMENT,
```

Comment:
Wind direction is a bit weird as it is technically a measurement, but will the statistics integration handle the correct circular statistics?

Reply:
I removed this as circular statistics are not supported